src/test/java/jewellery/inventory/unit/service/security/AuthServiceTest.java

```diff
@@ -0,0 +1,112 @@
+package jewellery.inventory.unit.service.security;
+
+import static jewellery.inventory.helper.UserTestHelper.USER_EMAIL;
+import static jewellery.inventory.helper.UserTestHelper.USER_PASSWORD;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import java.util.Optional;
+import java.util.UUID;
+import jewellery.inventory.dto.request.AuthenticationRequestDto;
+import jewellery.inventory.dto.response.UserAuthDetailsDto;
+import jewellery.inventory.dto.response.UserResponseDto;
+import jewellery.inventory.exception.not_found.UserNotFoundException;
+import jewellery.inventory.exception.security.InvalidCredentialsException;
+import jewellery.inventory.mapper.UserMapper;
+import jewellery.inventory.model.User;
+import jewellery.inventory.repository.UserRepository;
+import jewellery.inventory.security.JwtTokenService;
+import jewellery.inventory.service.AuthService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+
+@ExtendWith(MockitoExtension.class)
+class AuthServiceTest {
+  private static final String TEST_TOKEN = "token";
+  @InjectMocks private AuthService authService;
+  @Mock private UserRepository userRepository;
+  @Mock private AuthenticationManager authenticationManager;
+  @Mock private JwtTokenService jwtTokenService;
+  @Mock private UserMapper userMapper;
+
+  private AuthenticationRequestDto authRequest;
+  private User user;
+
+  @BeforeEach
+  public void setUp() {
+    authRequest = new AuthenticationRequestDto();
+    authRequest.setEmail(USER_EMAIL);
+    authRequest.setPassword(USER_PASSWORD);
+
+    user = new User();
+    user.setId(UUID.randomUUID());
+    user.setEmail(USER_EMAIL);
+    user.setPassword(USER_PASSWORD);
```

Comment:
Isn't this the same as UserTestHelper.createTestUser() ? Let's use it.

Reply:
Fixed